ovs-p4rt/ovs_p4rt_session.cc

```diff
@@ -13,6 +13,14 @@
 #include "grpcpp/create_channel.h"
 #include "p4/v1/p4runtime.grpc.pb.h"
 #include "p4/v1/p4runtime.pb.h"
+#include "stratum/glue/status/status.h"
+#include "stratum/glue/status/status_macros.h"
+#include "stratum/lib/utils.h"
+#include "stratum/public/proto/p4_role_config.pb.h"
+
+// TODO: Use ABS_FLAG for file location
+#define DEFAULT_OVS_P4RT_ROLE_CONFIG_FILE \
+  "/usr/share/stratum/networking_role_config.pb.txt"
```

Comment:
suggestion: ovs_p4rt_role_config.pb.txt?
In future, we might convert krnlmon calls to p4rt instead of TDI. So, it better to have module specific name instead of generic networking

Reply:
Updated